unlock-app/jest.config.js

```diff
@@ -8,8 +8,9 @@ module.exports = {
   ],
   transform: {
     '^.+\\.(js|jsx)$': 'babel-jest',
+    '^.+\\.(ts|tsx)$': 'ts-jest',
```

Comment:
Like eslint, Jest doesn't support TS by default. A package had to be added and some minor config changes. 
Nothing esoteric.